tools/bazel-external-tools.rc

```diff
@@ -1,4 +1,4 @@
 # generated
-build --deleted_packages=external-tools/buildtools/buildozer,external-tools/buildtools/build_proto,external-tools/buildtools/wspace,external-tools/buildtools/deps_proto,external-tools/buildtools/buildifier2,external-tools/buildtools,external-tools/buildtools/unused_deps,external-tools/buildtools/extra_actions_base_proto,external-tools/buildtools/edit,external-tools/buildtools,external-tools/buildtools/build,external-tools/buildtools/differ,external-tools/buildtools/generatetables,external-tools/buildtools/tables,external-tools/buildtools/lang,external-tools/buildtools/config,external-tools/buildtools/file,external-tools/buildtools/buildifier,external-tools/buildtools/api_proto,
-query --deleted_packages=external-tools/buildtools/buildozer,external-tools/buildtools/build_proto,external-tools/buildtools/wspace,external-tools/buildtools/deps_proto,external-tools/buildtools/buildifier2,external-tools/buildtools,external-tools/buildtools/unused_deps,external-tools/buildtools/extra_actions_base_proto,external-tools/buildtools/edit,external-tools/buildtools,external-tools/buildtools/build,external-tools/buildtools/differ,external-tools/buildtools/generatetables,external-tools/buildtools/tables,external-tools/buildtools/lang,external-tools/buildtools/config,external-tools/buildtools/file,external-tools/buildtools/buildifier,external-tools/buildtools/api_proto,
-test --delete_packages=external-tools/buildtools/buildozer,external-tools/buildtools/build_proto,external-tools/buildtools/wspace,external-tools/buildtools/deps_proto,external-tools/buildtools/buildifier2,external-tools/buildtools,external-tools/buildtools/unused_deps,external-tools/buildtools/extra_actions_base_proto,external-tools/buildtools/edit,external-tools/buildtools,external-tools/buildtools/build,external-tools/buildtools/differ,external-tools/buildtools/generatetables,external-tools/buildtools/tables,external-tools/buildtools/lang,external-tools/buildtools/config,external-tools/buildtools/file,external-tools/buildtools/buildifier,external-tools/buildtools/api_proto,
+build --deleted_packages=
+query --deleted_packages=
+test --delete_packages=
```

Comment:
Did this cause an issue? It's here because we can add a buildifier as a separate WORKSPACE without having `...` pick up anything (and not have to include buildifier in the top WORKSPACE, because it gets messy with transitivity).

Reply:
I didn't specifically edit this file-- this is just whatever happened after I formatted the repo locally.

Reply:
Hm, okay. I'll take a look.